lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -162,6 +167,32 @@ def sale_records_info(sale, features):
     return [sale_records_info(sale, features) for sale in sales]
 
 
+def issued_certificates_features(course_key, features):
```

Comment:
Nit: I'm confused by this name. It sounds like it is returning features, but it is also taking features as an argument. Can you describe the parameters in the docstring and how they are used.